R/z_helperFunctions.R

```diff
@@ -1,41 +1,41 @@
-## General helper functions
-
-
-#' Convert \code{pt} value to \code{lines}
-#' 
-#' @param pt_value Value in \code{pt} to be converted to \code{lines}
-#' @return Value in \code{lines}
-#' @note Does NOT work if input is not in \code{pt}. Input is returned as is.
-pt.to.lines <- function(pt_value){
-  if(grid::unitType(pt_value) == "pt" || grid::unitType(pt_value) == "points"){
-    pt_value <- round(as.numeric(pt_value) * (0.25/5.5), digits = 2)
-  }
-  as.numeric(pt_value)
-}
-
-
-#' Check if character is an RGB hexadecimal color value
-#' @param x character
-#' @return True/False value
-#' @export
-is.rgb <- function(x){
-  if(is.null(x)) {
-    TRUE
-  } else {
-    (grepl("#", x) & nchar(x)==7)
-  }
-}
-
-#' Convert R colors to RGB hexadecimal color values
-#' @param x character
-#' @return hexadecimal color value or "transparent" if is.na
-#' @importFrom grDevices col2rgb rgb
-#' @export
-toRGB <- function(x){
-  is.transparent <- is.na(x) | x=="transparent"
-  rgb.mat <- col2rgb(x)
-  rgb.vec <- rgb(t(rgb.mat), maxColorValue=255)
-  named.vec <- ifelse(is.transparent, "transparent", rgb.vec)
-  not.named <- as.character(named.vec)
-  not.named
-}
+## General helper functions
+
+
+#' Convert \code{pt} value to \code{lines}
+#' 
+#' @param pt_value Value in \code{pt} to be converted to \code{lines}
+#' @return Value in \code{lines}
+#' @note Does NOT work if input is not in \code{pt}. Input is returned as is.
+pt.to.lines <- function(pt_value){
+  if(grid::unitType(pt_value) %in% c("pt", "points")){
+    pt_value <- round(as.numeric(pt_value) * (0.25/5.5), digits = 2)
+  }
+  as.numeric(pt_value)
+}
+
+
+#' Check if character is an RGB hexadecimal color value
+#' @param x character
+#' @return True/False value
+#' @export
+is.rgb <- function(x){
+  if(is.null(x)) {
+    TRUE
+  } else {
+    (grepl("#", x) & nchar(x)==7)
+  }
+}
+
```

Comment:
hey @lazycipher  did you edit this is.rgb function at all? I dont see any changes. (did the line endings change or something?)
if there are no significant changes (white space etc) then please do NOT commit, because it makes it more difficult to code review.

Reply:
I made changes in the pt.to.lines function.
It looks like there's some issue with the line endings!
I really don't understand why whole file shows as edited while I do change just few lines.
Even on VS Code, It was showing 1 line change before committing.

Reply:
you should always review the output of the "git diff" command before you do a "git commit" and make sure to remove any parts of the diff which are not relevant to the intended change you are trying to make.
In terms of editors I would recommend using Emacs for this project because there is good support for both R (via ESS) and JavaScript editing. Or try Rstudio if Emacs is too complicated for you to learn right now (but if you want to become a good programmer you should learn at least one of the two classic editors, emacs and vim)

Reply:
I'll make sure to follow this up from now on. I'll use Emacs from now on for this project.

Reply:
here are some videos I made about emacs + ess https://www.youtube.com/playlist?list=PLwc48KSH3D1Onsed66FPLywMSIQmAhUYJ

Reply:
Thank you so much! I'll go through this.